specification/billingbenefits/resource-manager/Microsoft.BillingBenefits/stable/2022-11-01/billingbenefits.json

```diff
@@ -0,0 +1,1912 @@
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.BillingBenefits/operations": {
+      "get": {
+        "summary": "Get operations.",
+        "description": "List all the operations.",
+        "operationId": "Operations_List",
+        "x-ms-examples": {
+          "OperationsGet": {
+            "$ref": "./examples/OperationsGet.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "tags": [
+          "Benefits operations"
+        ],
+        "responses": {
+          "200": {
+            "description": "List all the operations.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.BillingBenefits/savingsPlanOrderAliases/{savingsPlanOrderAliasName}": {
+      "put": {
+        "description": "Create a savings plan.",
+        "operationId": "SavingsPlanOrderAlias_Create",
+        "tags": [
+          "Savings plan order aliases"
+        ],
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "SavingsPlanOrderAliasCreate": {
+            "$ref": "./examples/SavingsPlanOrderAliasCreate.json"
+          },
+          "SavingsPlanOrderAliasCreateSingleScope": {
+            "$ref": "./examples/SavingsPlanOrderAliasCreateSingleScope.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SavingsPlanOrderAliasNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SavingsPlanOrderAliasRequestParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request is being processed.",
```

Comment:
long running operations MUST either return a 201 or 202 response

Reply:
> long running operations MUST either return a 201 or 202 response

Please refer to https://github.com/Azure/azure-resource-manager-rpc/blob/ef4b8c67c7a1d793b4d1e1171777797c4ceef55f/v1.0/Addendum.md#creatingupdating-using-put. Reason here being the resource will still be provisioning after the request is completed, it might go to "Created" state or "Failed" state eventually. 201 means "Created" and it does not apply to this async operation behavior, thus we are using 200 here.

Reply:
A long running PUT that supports creation of a resource should return a 201 as per ARM RPC, Azure API guidelines and the HTTP RFC